spark-connector/spark-connector-runtime/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ tasks.withType<ShadowJar>(ShadowJar::class.java) {
   relocate("com.google", "com.datastrato.gravitino.shaded.com.google")
   relocate("google", "com.datastrato.gravitino.shaded.google")
   relocate("org.apache.hc", "com.datastrato.gravitino.shaded.org.apache.hc")
+  relocate("org.apache.logging.slf4j", "com.datastrato.gravitino.shaded.org.apache.logging.slf4j")
```

Comment:
Since you already relocate the slf4j jar in client-java-runtime, do you need to relocate again in here?

Reply:
I checked by extracting the jar package, it doesn't exist, thanks for pointing this.